pkg/cloudprovider/providers/oci/ccm.go

```diff
@@ -219,7 +218,7 @@ func (cp *CloudProvider) HasClusterID() bool {
 }
 
 func buildConfigurationProvider(logger *zap.Logger, config *providercfg.Config) (common.ConfigurationProvider, error) {
```

Comment:
Are we able to do away with this as well now?

Reply:
Good spot. Thanks. Fixed.